sdk/keyvault/azure.keyvault.secrets.txt

```diff
@@ -0,0 +1,57 @@
+**** Package/Module azure.keyvault.secrets****
+class SecretClient(KeyVaultClientBase, object):
+	vault_url:str
+	credential
+
+	vault_url
+
+	def __init__(self, vault_url: str, credential, transport=None, api_version=None, **kwargs): ...
+	def backup_secret(self, name: str, **kwargs) -> bytes: ...
+	def delete_secret(self, name: str, **kwargs) -> ~azure.keyvault.secrets.models.DeletedSecret: ...
+	def get_deleted_secret(self, name: str, **kwargs) -> ~azure.keyvault.secrets.models.DeletedSecret: ...
+	def get_secret(self, name: str, version: str=None, **kwargs) -> ~azure.keyvault.secrets.models.Secret: ...
+	def list_deleted_secrets(self, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.secrets.models.DeletedSecret]: ...
+	def list_secret_versions(self, name: str, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.secrets.models.SecretProperties]: ...
+	def list_secrets(self, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.secrets.models.SecretProperties]: ...
+	def purge_deleted_secret(self, name: str, **kwargs): ...
+	def recover_deleted_secret(self, name: str, **kwargs) -> ~azure.keyvault.secrets.models.SecretProperties: ...
+	def restore_secret(self, backup: bytes, **kwargs) -> ~azure.keyvault.secrets.models.SecretProperties: ...
+	def set_secret(self, name: str, value: str, content_type: str=None, enabled: bool=None, not_before: datetime.datetime=None, expires: datetime.datetime=None, tags: dict=None, **kwargs) -> ~azure.keyvault.secrets.models.Secret: ...
```

Comment:
I guess "enabled: bool=None" is about not populating the model to send on the wire, but would be clearer IMHO to have "True/False" for real. Unless customer can change the default on the server side, then I can understand it's not possible to take side on the client side. But if server is always "False" by default, I don't see how Azure can possibly change one day, I would suggest to make it clear here.
At worst, update the doc to say "Azure default if nothing is set is False" or whatever the actual default value.

Reply:
We talked about this in other languages as well. I think it's possible that this is still nullable here but the model type is not. In this case the default on the service is True.

Reply:
As mentioned in the secrets review if the service were to change we can possibly patch it at a later time.